doc/CwlMapping.md

```diff
@@ -0,0 +1,37 @@
+# Cwl Mappings
+
+## CommandInputParameter
+
+* doc -> HelpAttribute
+* label -> LabelAttribute
+* secondaryFiles -> ?
+* format -> ?
+* streamable -> ?
+* loadContents -> ?
+* loadListings -> ?
+
+* output.name -> name
+* output.types -> dxType
+* input.outputBinding -> ??
+
+
+## CommandOutputParameter
+
+* doc -> HelpAttribute
+* label -> LabelAttribute
+* secondaryFiles -> ?
+* format -> ?
+* streamable -> ?
+* loadContents -> ?
+* loadListings -> ?
+
+* output.name -> name
+* output.types -> dxType
+* output.outputBinding -> ??
+
+## questions
+
+* WDL logger is in wdltools, how about cwl logger?
+* Extras are used to create IR application - I.e. InstanceType / container, correct? These are also given when starting the app
```

Comment:
No, extras are only given at compile time. Any information that is needed at runtime is added to the generated applet's `details` metadata.